fitnessllm_dataplatform/stream/strava/services/etl_interface.py

```
@@ -0,0 +1,245 @@
+            ]
+            dataframe = DataFrame(metrics_list)
+            self.client.load_table_from_dataframe(
+                dataframe=dataframe,
+                destination=destination,
+                job_config=bigquery.LoadJobConfig(
+                    write_disposition=bigquery.WriteDisposition.WRITE_APPEND
+                ),
+            )
+        except Exception as e:
+            logger.error("Unable to write metrics to BigQuery.")
+            raise e
```

Ruff (0.8.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)